core/trino-main/src/main/java/io/trino/server/testing/TestingTrinoServer.java

```diff
@@ -139,6 +141,11 @@
 public class TestingTrinoServer
         implements Closeable
 {
+    static {
+        Logging logging = Logging.initialize();
+        logging.setLevel("io.trino.event.QueryMonitor", Level.OFF);
```

Comment:
We should make all these WARN or ERROR unless we need absolutely need them to be OFF.

Reply:
I'll see what happens with WARN or ERROR, but in general, the problem is that many parts of the code generate logs at those levels for recoverable events, and in situations encountered normally (e.g., server shutdown, etc).